core/src/Cuda/Kokkos_Cuda.hpp

```diff
@@ -54,7 +54,6 @@ class CudaInternal;
 namespace Kokkos {
 
 namespace Impl {
-namespace Experimental {
 enum class CudaLaunchMechanism : unsigned {
```

Comment:
It's probably worth mentioned in the PR description that you're moving this out of `Experimental` ?

Reply:
`Impl::` denote private implementation details that users are not allowed to spell in the code they write, so it makes little sense to nest `Experimental::` which is generally meant to make users aware something might be less mature and/or subject to change.
You will find that pattern at places in our code base (often in experimental backends) but we discourage doing it in general.

Reply:
And in case that was not obvious by dropping it, we are able to get rid of mouthful qualifications all over the place.